.github/workflows/kind-cluster-image-policy.yaml

```diff
@@ -160,10 +160,12 @@ jobs:
     - name: Generate New Signing Key
       run: |
         COSIGN_PASSWORD="" ./cosign generate-key-pair
+        mv cosign.key cosign-key.key
```

Comment:
@DennyHoang  Is this intentional ?

Reply:
Yes, for now, I was going to run `generate-key-pair` multiple times in the workflow file. 
So I move the cosign.key and pub after creation.

There is a lower step where I generate for the remote signing key

Reply:
Umm, okay. But I don't see cosign.pub been used later in this script.

Reply:
Oh, all the cosign.pub were renamed as either "cosign-remote.pub" or "cosign-key.pub" to match their key counterpart. 

Maybe the name for "cosign-key" file pairing was a bad choice. I could be more descriptive with "cosign-colocated-signing.key" and for the remote one, "cosign-remote-signing.key" and their pub respectively.

One key designated for having signature colocated with image, and the second key for signing with signatures in remote location.

Reply:
resolving as I did update those names as mentioned